core/arch/aarch64/encode_gen.h

```diff
@@ -5986,6 +5986,24 @@ encode_opndsgen_d61f0000(byte *pc, instr_t *instr, uint enc)
 }
 
 static uint
+encode_opndsgen_d63f0000(byte *pc, instr_t *instr, uint enc)
+{
+    int opcode = instr->opcode;
+    uint dst0, src0;
+    if (instr_num_dsts(instr) == 1 && instr_num_srcs(instr) == 1 &&
+        encode_opnd_impx30(enc & 0xfffffc1f, opcode, pc, instr_get_dst(instr, 0), &dst0) &&
+        encode_opnd_x5(enc & 0xffffffff, opcode, pc, instr_get_src(instr, 0), &src0)) {
+        ASSERT((dst0 & 0xffffffff) == 0);
+        ASSERT((src0 & 0xfffffc1f) == 0);
+        enc |= dst0 | src0;
```

Comment:
That's automatically generated by core/arch/aarch64/codec.py. If we want to fix this style issue, I could update the codec.py script and push an updated version of encode_gen.h